forge-gui/res/cardsfolder/upcoming/white_suns_twilight.txt

```diff
@@ -0,0 +1,10 @@
+Name:White Sun's Twilight
+ManaCost:X W W
+Types:Sorcery
+A:SP$ GainLife | LifeAmount$ X | SubAbility$ DBToken | SpellDescription$ You gain X life. Create X 1/1 colorless Phyrexian Mite artifact creature tokens with toxic 1 and "This creature can't block." If X is 5 or more, destroy all other creatures. (Players dealt combat damage by a creature with toxic 1 also get a poison counter.)
```

Comment:
Some of this SpellDescription needs to be moved to other lines, or you need to make use of StackDescription stuff

Ensure that the stack description when this is cast doesn't have repeated or confusing lines